Demo/src/main/java/com/braintreepayments/demo/PayPalNativeFragment.java

```diff
@@ -4,29 +4,99 @@
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.Button;
 
+import androidx.fragment.app.FragmentActivity;
+import androidx.lifecycle.ViewModelProvider;
+import androidx.navigation.fragment.NavHostFragment;
+
+import com.braintreepayments.api.BrowserSwitchResult;
+import com.braintreepayments.api.DataCollector;
+import com.braintreepayments.api.PayPalClient;
+import com.braintreepayments.api.PayPalNativeCheckoutRequest;
 import com.braintreepayments.api.PayPalNativeClient;
+import com.braintreepayments.api.PaymentMethodNonce;
+
+import static com.braintreepayments.demo.PayPalRequestFactory.createPayPalCheckoutRequest;
+import static com.braintreepayments.demo.PayPalRequestFactory.createPayPalNativeVaultRequest;
 
 public class PayPalNativeFragment extends BaseFragment {
 
-    PayPalNativeClient payPalNativeClient;
+    private PayPalNativeClient payPalNativeClient;
+
+    private String deviceData;
+    private DataCollector dataCollector;
+
+
+    private Button billingAgreementButton;
```

Comment:
I know we haven't updated the `PayPalFragment` in the demo app for this, but let's go with `vault` and `checkout` for the variable/method naming throughout to align with the request types